server/src/test/java/org/elasticsearch/action/bulk/BulkRequestTests.java

```diff
@@ -219,20 +219,6 @@ public void testBulkRequestWithRefresh() throws Exception {
                 "RefreshPolicy is not supported on an item request. Set it on the BulkRequest instead."));
     }
 
-    // issue 15120
-    public void testBulkNoSource() throws Exception {
```

Comment:
I just removed this test, adding a request without a source isn't possible any longer and now trips an assertion.

Reply:
Which assertion is tripped? I would think it still makes sense to verify that we have the errors below come out if someone should use this through the transport client in 7.x? Also maybe worth checking that we have this case covered in a REST test.

Reply:
Sorry for the noise here, this test works just fine still. It was incompatible with an earlier version of this change but with the current version it again works as it always has => I put it back :)